### ISSUE TITLE
Exported pages bigger than source pages

### ISSUE THREAD

**Issue by tuxedotje** [pdfarranger/pdfarranger#676]
Hi, thanks for pdfarranger. 
Describe the problem (this is a wish)
When exporting single pages the total of kB' of the exported pages is higher than the kB s of the source.
I do not know if this is a bug.
But exporting pages of a big source did give very large pdf's.
Would be great to have smaller files.
An example of very big pages after export.
a source file 22.7 MB (124 pages) split into single files (or files with 1 or 2 pages). Most of the exported files was 22.6 MB, some had about 200 kB. 
A source file 132 kB (2 pages) exported to 2 files each of 70.0 kB (single page)
What to do about this?

**Comment by jeromerobert** [comment]
> One could investigate this further to figure out whether the files exported by PDF Arranger are the absolute minimum

They are not. If you import a PDF file with one big image multiple times you would like to have this image deduplicated when exporting but, unfortunately, we don't do that. This is beyond what PDF Arranger can do because it would require to parse the content of each pages.

**Comment by dreua** [comment]
> [...] If you import a PDF file with one big image multiple times you would like to have this image deduplicated when exporting but, unfortunately, we don't do that. This is beyond what PDF Arranger can do because it would require to parse the content of each pages.

Good point, this issue seems to be focused on splitting files, not combining them. In that context I'm not aware of any issues on creating unnecessarily big files. The de-duplication would be a nice feature though, beyond the scope of PDF Arranger but maybe in scope of pikepdf or QPDF.

**Comment by tuxedotje** [comment]
Thanks for your comments.
Thanks for your answers.
It is not the pdfarranger that causes some troubles.
I do  send these exported pdf files by email and that sometimes is a problem with big files.
From now I will use other means to send pdf files.
Thanks again,  Jaap
